tests/containers/helm.pm

```diff
@@ -26,8 +26,9 @@ sub run {
     my ($self, $run_args) = @_;
 
     # We either test various cloud clusters or local k3s
-    my $k8s_backend = get_required_var('HELM_K8S_BACKEND_2');
+    my $k8s_backend = $run_args->{backends}->[0];
```

Comment:
I was thinking to do the `shift` over here :) Instead of `main_containers` as you need to preserve the backends until test runtime

Reply:
Works, thanks!